### PR TITLE
Boost pr

### PR DESCRIPTION
**Pull request by 960**
Here goes!

I am not able to split this up, it was hard enough making a clean uplopad.

Have a look whenever you find time, and feel free to edit if you find anything to improve.

### CODE REVIEW

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

**Comment by rusefi** [comment]
@960 your rusefi_config.txt was negatively affected by https://github.com/rusefi/rusefi/issues/1115 defect

I've fixed the defect and accepted some first files into master. Please rebase your branch (please google "git rebase") - https://github.com/960/rusefi/tree/boost_pr should NOT be saying "131 commits behind rusefi:master."

**Comment by rusefi** [comment]
add to while list

File: firmware/integration/rusefi_config.txt [hunk 1]
```diff
@@ -744,15 +744,20 @@ custom maf_sensor_type_e 4 bits, S32, @OFFSET@, [0:7], @@maf_sensor_type_e_enum@
 
     output_pin_e boostControlPin;
 	pin_output_mode_e boostControlPinMode;
+
```
Comment (rusefi) [inline firmware/integration/rusefi_config.txt]: not useful changes in rusefi_config.txt so please revert this file - this would revert it from PR

Comment (960) [reply firmware/integration/rusefi_config.txt]: I only edited something to resolve conflict.. How do I revert it?

Have tried to delete it as well..

**Comment by 960** [comment]
This one should be ready now I belive..

**Pull request merged**